src/Organizations/OrganizationForm/OrganizationSummaryForm/OrganizationSummaryForm.js

```diff
@@ -143,6 +174,22 @@ function OrganizationSummaryForm({ initialValues }) {
           ))}
         </Field>
       </Col>
+      <Col
+        xs={6}
+        md={3}
+      >
+        <FieldMultiSelectionFinal
+          ariaLabelledBy="organizationFormTypesLabel"
+          dataOptions={typeOptions}
+          filter={filter}
+          formatter={formatter}
+          id="organizations-type"
+          itemToString={itemToString}
+          label={<FormattedMessage id="ui-organizations.summary.type" />}
+          name="organizationTypes"
+          onChange={getSelectedTypes}
```

Comment:
it doesn't look right when onChange calls getSelectedTypes - change usually doesn't get

Reply:
changed to `const { change } = useForm();`